src/ontology/mondo-edit.obo

```diff
@@ -574661,6 +574661,17 @@ is_a: MONDO:0022800 {source="https://clinicalgenome.org/affiliation/40065/", sou
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7379" xsd:anyURI
 
+[Term]
+id: MONDO:1040002
+name: developmental and epileptic encephalopathy, 31A
+subset: nord_rare {source="MONDO:NORD"}
```

Comment:
@yshwetar we do not add the rare subclasses manually. These will be added automatically. However, once the PR is approved and merged, please make sure to let Gioconda know that we created this term, and give her the ID, so she can update her NORD list. We can work on the wording together. 
We have not discussed the rare disease subset. We can add it to the list of things to review together.